<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>拍手游戏</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>690</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="mainPage">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>420</y>
        <width>95</width>
        <height>139</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainPage_VLayout">
       <item>
        <widget class="QPushButton" name="startGameButton">
         <property name="text">
          <string>开始游戏</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="settingsButton">
         <property name="text">
          <string>设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="helpButton">
         <property name="text">
          <string>帮助</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitGameButton">
         <property name="text">
          <string>退出游戏</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="startGamePage">
     <widget class="QPushButton" name="startToMainButton">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>530</y>
        <width>130</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>200</y>
        <width>271</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="startGamePage_VLayout">
       <item>
        <widget class="QPushButton" name="mode1Button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>模式1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mode2Button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>模式2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mode3Button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>模式3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mode4Button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>模式4</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="settingsPage">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <widget class="QPushButton" name="settingsToMainButton">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>530</y>
        <width>130</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>291</y>
        <width>421</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="sound_HLayout">
       <item>
        <widget class="QLabel" name="soundLabel">
         <property name="text">
          <string>音效 </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="soundshowLabel">
         <property name="text">
          <string>50</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sound_HSlider">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>350</y>
        <width>421</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="BGM_HLayout">
       <item>
        <widget class="QLabel" name="BGMLabel">
         <property name="text">
          <string>BGM</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="BGMshowLabel">
         <property name="text">
          <string>50</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="BGM_HSlider">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="pageStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="sliderPosition">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="helpPage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>180</y>
        <width>81</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>帮助页面</string>
      </property>
     </widget>
     <widget class="QPushButton" name="helpToMainButton">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>530</y>
        <width>130</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="playGamePage">
     <widget class="QPushButton" name="pauseButton">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>20</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>暂停游戏</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pausePage">
     <widget class="QPushButton" name="backToGameButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>330</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>返回游戏</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>400</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>退出游戏</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_O"/>
    <addaction name="action_S"/>
    <addaction name="separator"/>
    <addaction name="action_X"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_rules"/>
    <addaction name="separator"/>
    <addaction name="action_v1_0_1"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="action_X">
   <property name="text">
    <string>关闭(&amp;X)</string>
   </property>
  </action>
  <action name="action_v1_0_1">
   <property name="text">
    <string>拍手游戏v1.0.0</string>
   </property>
   <property name="toolTip">
    <string>拍手游戏v1.0.0</string>
   </property>
  </action>
  <action name="action_O">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="action_S">
   <property name="text">
    <string>另存为(&amp;S)</string>
   </property>
  </action>
  <action name="action_rules">
   <property name="text">
    <string>游戏规则</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
